docs/developer-guide/site.md

```diff
@@ -7,19 +7,15 @@ The website is build using `mkdocs` and `mkdocs-material`.
 To test:
 
 ```bash
+make build-docs
 make serve-docs
 ```
-Once running, you can view your locally built documentation at [http://0.0.0.0:8000/](http://0.0.0.0:8000/). 
 
-## Deploying
-
-```bash
-make publish-docs
-```
```

Comment:
This target doesn't seem to exist anymore.

Also, this doc might benefit from a link to the contribution guidelines, which could themselves use a section about documentation-only contributions. I didn't add those here since they're a bit far afield of my original aim of updating the Okta docs.